presto-product-tests/bin/run_on_docker.sh

```diff
@@ -85,10 +85,10 @@ function cleanup() {
   # In most cases after the docker containers are stopped, logs processes must be terminated.
   # However when the `LEAVE_CONTAINERS_ALIVE_ON_EXIT` is set, docker containers are not being terminated.
   # Redirection of system error is supposed to hide the `process does not exist` and `process terminated` messages
-  if [[ ! -z ${HADOOP_LOGS_PID} ]]; then
```

Comment:
The same construct is used in `./presto-product-tests/bin/lib.sh:18`, it looks like it should be also fixed.

Is it all about that macos does not support this syntax? If so then commit message is a bit misleading.

Reply:
Nope. The code before the change was testing if `${HADOOP_LOGS_PID}` is non-empty, so it was failing when `HADOOP_LOGS_PID` variable was not defined (yet).

The comment about Mac explains (aims to explain) why i didn't do the fix "properly", which is using `test -v` which tests just that: if a variable is defined.

Reply:
> The same construct is used in ./presto-product-tests/bin/lib.sh:18, it looks like it should be also fixed.

This is different, since the variable used there is assigned in the preceeding line.